src/lib/parse/CompoundValueNode.h

```diff
@@ -127,6 +126,7 @@ class CompoundValueNode
   void                check(void);
   std::string         finish(void);
   std::string         render(Format format, const std::string& indent);
+  std::string         renderV2(void);
```

Comment:
What if we call it 'toJson()' ?
Later on, if changes are needed from v2 to v3, a parameter can be added (version).